### PR TITLE
f-error-boundary@v0.1.0 - Initial generated component.

### PR DESCRIPTION
**Pull request by DamianMullins**
Component, documentation etc to follow.

### Added
- Initial generated component

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -25135,6 +25250,18 @@ vite@2.4.4:
   optionalDependencies:
     fsevents "~2.3.2"
 
+vite@2.6.9:
```
Comment (jamieomaguire) [inline yarn.lock]: I didn't realise we could use vite with fozzie components yet!

**Comment by jamieomaguire** [comment]
is this going to behave similar to the version we use on SW?

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/atoms/f-error-boundary/src/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (ashleynolan) [inline packages/components/atoms/f-error-boundary/src/index.js]: Potentially stupid q – is this a component, or is it more of a service? (just as I kind of expect atoms to have a `.vue` style SFC)

File: yarn.lock (hunk 1 above)
Comment (ashleynolan) [reply yarn.lock]: We can use it for JS services – for anything using SFC structure, we're going to look at moving to Vite shortly when we update to Vue 3 (as we need to look at what things we rely on that may need to be handled by Vite)

Comment (DamianMullins) [reply yarn.lock]: We'll be able to vite for this component as it has no dependencies on SCSS. I'll post a draft PR for discussion after this one.

File: packages/components/atoms/f-error-boundary/src/index.js (hunk 2 above)
Comment (DamianMullins) [reply packages/components/atoms/f-error-boundary/src/index.js]: It'll be a renderless component (in a Vue file), I kept this file here for the initial commit but it'll be removed in the next PR.

**Comment by DamianMullins** [comment]
@jamieomaguire very similar, yes. Hoping to offer a few different ways of reporting the errors whilst also being configurable.

File: packages/components/atoms/f-error-boundary/src/index.js (hunk 2 above)
Comment (DamianMullins) [reply packages/components/atoms/f-error-boundary/src/index.js]: @ashleynolan I've updated the PR so that it now references a Vue component.

**Review by ashleynolan — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**